rpcs3/Emu/Cell/Modules/cellMusicSelectionContext.cpp

```diff
@@ -145,7 +145,7 @@ void music_selection_context::create_playlist(const std::string& new_hash)
 
 	fs::pending_file file(yaml_path);
 
-	if (!file.file || file.file.write(out.c_str(), out.size() < out.size() || !file.commit()))
+	if (!file.file || file.file.write(out.c_str(), out.size()) < out.size() || !file.commit())
```

Comment:
You do not need to check this. Truncated writes would result in an error which we check inside the function.